spec/factories/chargeback_rate.rb

```diff
@@ -11,4 +11,48 @@
       chargeback_rate.chargeback_rate_details << FactoryGirl.create(:chargeback_rate_detail_memory_allocated_with_tiers)
     end
   end
+
+  factory :chargeback_rate_with_compute_details, :parent => :chargeback_rate do
```

Comment:
[Although it's written like this in other factories, unfortunately,] when you see `with_blah` it usually means it should be rewritten as a [FactoryGirl traits](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits) within the `chargeback_rate` factory. It's a very quick change, so let's do that. Should be able to do `FactoryGirl.create(:chargeback_rate, :with_compute_details)`